packages/colorpickers/demo/colorSwatchDialog.stories.mdx

```diff
@@ -7,7 +7,12 @@ import { COLOR_SWATCH_COLORS as COLORS } from './stories/data';
 <Meta
   title="Packages/Colorpickers/ColorSwatchDialog"
   component={ColorSwatchDialog}
-  args={{ buttonProps: { 'aria-label': 'Label' }, colors: COLORS, isAnimated: true }}
+  args={{
+    buttonProps: { 'aria-label': 'Label' },
+    'aria-label': 'Storybook color swatch picker',
```

Comment:
[nit] ditto; recommend simplifying to "Title" (also the component is not a "color swatch picker" 😉 )

Reply:
what about a color swatch dialog picker panel? :P